### ISSUE TITLE
Is it really necessary to use `unix` there?

### ISSUE THREAD

**Issue by DuckerMan** [miquels/webdav-handler-rs#4]
https://github.com/miquels/webdav-handler-rs/blob/0acc45beda2ab42b022a8e9ef141edfc32278c78/src/localfs.rs#L12

**Comment by miquels** [comment]
Yes, because of https://github.com/miquels/webdav-handler-rs/blob/0acc45beda2ab42b022a8e9ef141edfc32278c78/src/localfs.rs#L336
